### DISCUSSION TITLE
Welcome to speech-recognition-uk Discussions!

### DISCUSSION THREAD

**Discussion by egorsmkv** [egorsmkv/speech-recognition-uk#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by asukhariev** [comment]
How can i use it without any ML knowledge? I want to make some voice to text (a lot) , can i do it any easy user friendly way like uploading text somewhere and get the actual text ? 
Thanks

**Reply by egorsmkv** [reply]
Hello. You can use Huggingface for it, they have such a feature. Just find HF links under the Implementations section.
